lib/modules/manager/npm/post-update/index.ts

```diff
@@ -242,7 +242,12 @@ export async function writeUpdatedPackageFiles(
       await writeLocalFile(packageFile.path, packageFile.contents!);
       continue;
     }
-    if (!packageFile.path.endsWith('package.json')) {
+    if (
+      !(
+        packageFile.path.endsWith('package.json') ||
+        packageFile.path.endsWith('pnpm-workspace.yaml')
```

Comment:
The hardcoding here tripped me up, I kept not seeing the lockfiles changing 😅 Does this not beat some of the point of allowing `pnpm-workspace.yaml` (and `package.json`) being renamed via `fileMatch`? I wonder what the original context for the check was.

Reply:
K, I think this will lead to too much changes in this PR. :/

Let's support `pnpm-workspace.yaml` and then support more names as needed.

Reply:
Yeah, it turned out to be a bit complicated indeed 😅  Would you rather I keep `pnpm-workspace.yaml` in `fileMatch` still? It probably does not hurt to keep it there, just to avoid any more changes, but I'm happy to remove it as well.